tests/inference/test_modeling.py

```diff
@@ -123,6 +123,15 @@ def test_trust_remote_code(self):
         self.assertIsInstance(model.model, torch.jit._script.ScriptModule)
         self.assertIsInstance(model.config, PretrainedConfig)
 
+    @requires_neuronx
+    def test_save_compiler_intermediary_files(self):
+        with tempfile.TemporaryDirectory() as tempdir:
+            save_path = f"{tempdir}/neff"
+            _ = NeuronModelForSequenceClassification.from_pretrained(
+                self.MODEL_ID, export=True, compiler_workdir=save_path, **self.STATIC_INPUTS_SHAPES
+            )
+            self.assertTrue(os.path.isdir(save_path))
```

Comment:
Maybe also check there is at least one neff in it ?

Reply:
same.